pkg/globalnet/controllers/iptables/iface.go

```diff
@@ -100,18 +100,7 @@ func (i *ipTables) RemoveClusterEgressRules(subnet, snatIP, globalNetIPTableMark
 }
 
 func (i *ipTables) ipTableChainExists(table, chain string) (bool, error) {
-	existingChains, err := i.ipt.ListChains(table)
-	if err != nil {
-		return false, errors.Wrapf(err, "error listing chains in IP table %q", table)
-	}
-
-	for _, val := range existingChains {
-		if val == chain {
-			return true, nil
-		}
-	}
-
-	return false, nil
+	return (i.ipt.ChainExists(table, chain))
```

Comment:
```suggestion
	return i.ipt.ChainExists(table, chain)
```

Reply:
I think we need the extra parentheses here,  to avoid  [1] lint error

[1] 
error returned from interface method should be wrapped: sig: func (github.com/submariner-io/submariner/pkg/iptables.Basic).ChainExists(table string, chain string) (bool, error) (wrapcheck)go-golangci-lint

Reply:
Urgh, that’s not a great workaround — don’t hide linting errors, disable them with a nolint comment if appropriate or address them.

In this instance, it would be better to drop the `ipTableChainExists` function entirely, since it’s only used once; and wrap the error on return from `GetKubeProxyClusterIPServiceChainName` as is done in other functions.